circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Looks like `8.3` already do that:
<img width="1112" alt="screen shot 2017-04-25 at 08 07 10" src="https://cloud.githubusercontent.com/assets/885516/25382505/4e3bc9fc-298e-11e7-889b-4f23d23495ea.png">
From https://circleci.com/gh/SwiftGen/StencilSwiftKit/119